### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by igor-ramazanov**
Add a section about `alloy.proto#grpc`.

I have been struggling a bit with this piece of a valuable information.

Not sure it is a right place to add or if it is written well, but I would like to see it somewhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -319,6 +319,50 @@ The following shapes are provided as a means of customizing how your Smithy shap
 - alloy.proto#protoReservedFields
 - alloy.proto#uncheckedExamples
 
+#### ⚠️ We do not provide out-of-the-box support neither for `protobuf` or `gRPC`.
```
Comment (Baccata) [inline README.md:322]:
```suggestion
#### ⚠️ We do not provide out-of-the-box support neither for `protobuf` or `gRPC` in smithy4s.
```

**Review by Baccata — COMMENTED** [review]
Haha, thanks for this 😄 . 

`alloy` is meant to be decoupled from (or unaware of, so to say) smithy4s, as we have other tools internally making us of it that aren't Scala specific. 

I'm not opposed to the mention, but we need to assume that the reader doesn't necessarily know about smithy4s, and contextualise the statement in consequence

File: README.md [hunk 2]
```diff
@@ -319,6 +319,50 @@ The following shapes are provided as a means of customizing how your Smithy shap
...
+    libraryDependencies ++= Seq(
+      "com.disneystreaming.smithy4s" %% "smithy4s-http4s" % smithy4sVersion.value,
+      "com.disneystreaming.smithy4s" %% "smithy4s-http4s-swagger" % smithy4sVersion.value,
+      "org.http4s" %% "http4s-ember-server" % "0.23.16"
+    ),
+    Compile / run / fork := true,
+    Compile / run / connectInput := true
+  )
+
+// Generate the hints from alloy.proto in a separate project. Their are not pre-packaged in smithy4s,
```
Comment (daddykotex) [inline README.md:351]:
```suggestion
// Generate the hints from alloy.proto in a separate project. They are not pre-packaged in smithy4s,
```

**Comment by igor-ramazanov** [comment]
>I'm not opposed to the mention, but we need to assume that the reader doesn't necessarily know about smithy4s, and contextualise the statement in consequence

Thanks, everything makes a sense now. `alloy` is a collection of common Smithy shapes, but how exactly they are processed - a totally different story...

File: README.md (hunk 2 above)
Comment (Baccata) [reply README.md]: lol that one's on me

**Comment by Baccata** [comment]
> alloy is a collection of common Smithy shapes, but how exactly they are processed - a totally different story... 

Exactly

**Pull request merged**